hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OmConfig.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.om;
+
+import org.apache.hadoop.hdds.conf.Config;
+import org.apache.hadoop.hdds.conf.ConfigGroup;
+import org.apache.hadoop.hdds.conf.ConfigTag;
+import org.apache.hadoop.hdds.conf.PostConstruct;
+
+/**
+ * Ozone Manager configuration.
+ */
+@ConfigGroup(prefix = "ozone.om")
+public class OmConfig {
```

Comment:
Nit: Maybe `OMConfig` is better (from `OMConfigKeys`)? However, the naming is not really enforced in the codebase, so I'm fine letting this be.

Reply:
There are examples of both spellings, e.g. `OmMetadataManager`, `OmUpgradeAction`, etc.

I think we are kind of treating OM as a word, not as an abbreviation.